rspace/src/main/scala/coop/rchain/rspace/ReplayRSpace.scala

```diff
@@ -315,6 +315,15 @@ class ReplayRSpace[F[_]: Sync, C, P, A, K](
     comms.tailRecM(go).map(_.toOption)
   }
 
+  def spawn: F[IReplaySpace[F, C, P, A, K]] = {
```

Comment:
I think this method should be in `RuntimeManager`.
I don't actually like spawn here. Maybe it is just personal favor. @tgrospic talked about this manager idea. I am more for that